### PR TITLE
Add changelog and changelog generation script

### PR DESCRIPTION
**Pull request by kabirbaidhya**
* Generate `CHANGELOG.md`
* Use [github_changelog_generator](https://github.com/skywinder/github-changelog-generator) to generate/update changelog
* Add script to make this easier.

Now you can do this to generate the markdown:
```
$ NEXT=0.0.5 yarn changelog
```
Or,
```
$ NEXT=0.0.5 ./t changelog
```

### CODE REVIEW

**Review by mesaugat — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -172,5 +172,8 @@ For more information [click here](https://developer.mozilla.org/en-US/docs/Web/A
 ## Contributing
 All kinds of contributions are welcome. Please feel free to propose PRs, report issues and suggestions to improve.
 
+## Change Log
```
Comment (mesaugat) [inline README.md]: I think `Changelog` is a single word.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Change Log
```
Comment (mesaugat) [inline CHANGELOG.md:1]: It should be a single word. Is this autogenerated? We need to send in a PR. 😛

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+## [v0.0.2](https://github.com/kabirbaidhya/keycode-js/tree/v0.0.2) (2016-04-14)
+[Full Changelog](https://github.com/kabirbaidhya/keycode-js/compare/v0.0.2-0...v0.0.2)
+
+## [v0.0.2-0](https://github.com/kabirbaidhya/keycode-js/tree/v0.0.2-0) (2016-04-05)
+[Full Changelog](https://github.com/kabirbaidhya/keycode-js/compare/v0.0.1-0...v0.0.2-0)
+
+## [v0.0.1-0](https://github.com/kabirbaidhya/keycode-js/tree/v0.0.1-0) (2016-04-04)
+
+
+\* *This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
```
Comment (mesaugat) [inline CHANGELOG.md:22]: Same here single word.

File: CHANGELOG.md (hunk 2 above)
Comment (kabirbaidhya) [reply CHANGELOG.md:1]: Yes, this is auto-generated file. Here it says so.
*This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*

File: CHANGELOG.md (hunk 3 above)
Comment (kabirbaidhya) [reply CHANGELOG.md:22]: Not possible right now, unless [they](https://github.com/skywinder/Github-Changelog-Generator) make it so :smile: .

**Pull request merged**